content/en/apps/tutorials/app-forms.md

```diff
@@ -0,0 +1,125 @@
+| ----------- | --------------- | --------------------------- | -------- | ----------------- | ---------- | ---------- | ------------------- | ---------------------- | -------------- | ----------------------------- | ------- |
+| begin group | inputs          | Patient                     |          | ./source = 'user' | field-list |            |                     |                        |                |                               |         |
+| hidden      | source          | Source                      |          |                   |            |            |                     |                        |                |                               | user    |
+| hidden      | source_id       | Source_ID                   |          |                   |            |            |                     |                        |                |                               |         |
+| begin group | contact         | Contact                     |          |                   |            |            |                     |                        |                |                               |         |
+| db:person   | _id             | Patient ID                  |          |                   | db-object  |            |                     |                        |                | Select a person from the list |         |
+| hidden      | patient_id      | Medic ID                    |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | name            | Patient Name                |          |                   |            |            |                     |                        |                |                               |         |
+| begin group | parent          | Parent                      |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | _id             | Family UUID                 |          |                   |            |            |                     |                        |                |                               |         |
+| begin group | parent          | Grandparent                 |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | _id             | CHW Area UUID               |          |                   |            |            |                     |                        |                |                               |         |
+| hidden      | name            | CHW Name                    |          |                   |            |            |                     |                        |                |                               |         |
```

Comment:
Missed `| begin group | contact          | CHW  |` that has CHW name and phone.

Reply:
Intentionally left them out to try and reduce the number of rows and simplify the table.